perf/impl/nim-libp2p/v1.0/Makefile

```diff
@@ -0,0 +1,21 @@
+commitSha := e03547ea3e2f0372c540c586e993803299d3c4b6
+
+all: perf
+
+perf: perf.nim nim-libp2p
+	docker run --rm --user "$(shell id -u):$(shell id -g)" -v "$(shell pwd)":/usr/src/myapp -w /usr/src/myapp nimlang/nim:1.6.10 sh -c "cd nim-libp2p && nimble install_pinned && cd - && nim c --NimblePath:nim-libp2p/nimbledeps/pkgs -p:nim-libp2p -d:chronicles_log_level=WARN --threads:off -d:release perf.nim"
```

Comment:
CI has failed:

```
bash: line 1: ./impl/nim-libp2p/v1.0/perf: No such file or directory
```

Is this actually producing a binary under the name `perf`?

Reply:
That's what I was afraid of. It's a Docker problem, it creates an ELF executable, no matter what, where it should create a host-compatible executable.

But it seems that there's no nim image on docker hub with this option.

Reply:
After a bit of research, the problem doesn't come from the executable itself, but come from the linking of the .so...
```
$ file perf
perf: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=895de8bd9fdf766ac9448fa05056cb4370bb6f54, for GNU/Linux 3.2.0, not stripped
```
/lib64/ld-linux-x86-64.so.2 isn't found

And I have no idea what I should use instead.